### PR TITLE
Md 77 set interval for style sheet

### PR DESCRIPTION
**Pull request by ThatOneGoat**

### CODE REVIEW

File: js/content.js [hunk 1]
```diff
@@ -23,6 +24,8 @@ let checkElement = document.createElement("div");
 var img = document.createElement("img");
 let m_Stream = null;
 
+let intervalCSSId;
```
Comment (mmattikalli) [inline js/content.js]: comment telling it what it does (like the rest of the variables above, for consistency)

File: js/content.js [hunk 2]
```diff
@@ -81,7 +85,9 @@ browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
             }
             break;
         case "Blur":
+            console.log("changed");
```
Comment (mmattikalli) [inline js/content.js]: debug print to be removed

File: js/content.js [hunk 3]
```diff
@@ -118,6 +124,9 @@ browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
                 }
             }, 1500);
             break;
+        case "AlertSlouch":
```
Comment (mmattikalli) [inline js/content.js]: Unnecessary

File: js/facelock.js [hunk 4]
```diff
@@ -52,6 +52,13 @@ function facelockMessageListener(message, sender, sendResponse) {
                                 return;
                             }
 
+                            // console.log(JSON.stringify(response[0].faceRectangle));
```
Comment (mmattikalli) [inline js/facelock.js]: commented portions to be removed

File: js/content.js [hunk 5]
```diff
@@ -308,6 +317,18 @@ function fadeOut(element) {
...
 
+function setCSSInterval() {
+    console.log("setCSSInterval is running");
+    intervalCSSId = setInterval(function () {
+        newWebsiteDiv.style.filter = "blur(20px)";
+    }, 100);
+}
+
+function stopCSSInterval() {
+    console.log("timer off");
```
Comment (mmattikalli) [inline js/content.js]: debug print? if yu want it in the console permenantly, specify what it does //

File: js/content.js (hunk 3 above)
Comment (ThatOneGoat) [reply js/content.js]: you are wrong

Comment (ThatOneGoat) [reply js/content.js]: hahahaha

File: js/content.js [hunk 6]
```diff
@@ -308,6 +317,18 @@ function fadeOut(element) {
     }, 20);
 }
 
+function setCSSInterval() {
+    console.log("setCSSInterval is running");
```
Comment (Kaboomboom3) [inline js/content.js]: Remove debug print

File: js/facelock.js [hunk 7]
```diff
@@ -83,7 +90,7 @@ function facelockMessageListener(message, sender, sendResponse) {
                             }
                         });
                     });
-                }, 3000);
+                }, 5000);
```
Comment (Kaboomboom3) [inline js/facelock.js]: Why was this changed to 5 seconds?
